CHANGELOG.md

```diff
@@ -3,7 +3,57 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 1.3.0
+
+This release contains a mix of features, and bugfixes.
+
+### Event-based Decision Log Buffer ([#5724](https://github.com/open-policy-agent/opa/issues/5724))
```

Comment:
Not sure about this headline. "event-based" isn't very helpful ..

Reply:
> New buffer method for Decision Logs

🤔 maybe?

Reply:
Yeah, or `New Buffer Option for Decision Logs` maybe

Reply:
Ideally, we'd have some numbers to back up our claims... like, _this changes our hubbabubba metric by 5% in the common cases!_ but I don't suppose anyone got a benchmark lying around